WordPress/Classes/ViewRelated/Support/SupportChatBot/SupportChatBotViewController.swift

```diff
@@ -0,0 +1,40 @@
+import WebKit
+
+final class SupportChatBotViewController: WebKitViewController {
+    private let viewModel: SupportChartBotViewModel
+
+    init(viewModel: SupportChartBotViewModel) {
+        self.viewModel = viewModel
+        let configuration = WebViewControllerConfiguration(url: viewModel.url)
+        configuration.secureInteraction = true
+        super.init(configuration: configuration)
+    }
+
+    required init?(coder aDecoder: NSCoder) {
+        fatalError("init(coder:) has not been implemented")
+    }
+
+    func webView(_ webView: WKWebView, didFinish navigation: WKNavigation!) {
+        webView.evaluateJavaScript(createDocsBotInitCode(), completionHandler: { (result, error) in
+            if let error = error {
+                DDLogError("Failed to initialize docs bot code: \(error)")
+            }
+        })
+    }
+
+    /// Creating DocsBotAI JavaScript code so we could tweak configuration from within Swift code
+    /// https://docsbot.ai/docs/embeddable-chat-widget
+    private func createDocsBotInitCode() -> String {
+        """
```

Comment:
We can inject other properties as well as call a message from a callback this way

```
DocsBotAI.init({
  id: 'YOUR_ID_HERE',
  supportCallback: function (event, history) {
    event.preventDefault() // Optionally prevent default behavior opening the url.
    console.log(history) // Safely access the chat history.
    DocsBotAI.close() // Close the widget.
  },
  identify: {
    name: 'John Doe', // This will be shown in the chat history logs.
    email: 'john@doe.com', // This will be shown in the chat history logs.
    customVariable: 'customValue', // This will be recorded in the question metadata and accessible via the API.
  },
}).then(() => {
  // Safely do stuff here after the widget is loaded.
})
```
https://docsbot.ai/docs/embeddable-chat-widget

I couldn't indicate a better way.